src/components/Buttons.js

```diff
@@ -1,46 +1,92 @@
+/* eslint-disable react/prop-types */
 import React from 'react';
+import PropTypes from 'prop-types';
 
 class Buttons extends React.PureComponent {
   render() {
+    const { state, buttonClicked } = this.props;
+    const { total, next, operation } = state;
     return (
       <>
-        <div className="results"> 0</div>
+        <div className="results">
+          {total}
+          {operation}
+          {next}
+        </div>
 
         <div className="mathOperations">
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/211905841-50ff35bc-afed-4058-933f-724816f96404.png)

- [ ] This link is not working. Kindly fix this link to make your readme file descriptive and professional :+1:

**Hint: You don't have a LICENSE file in your repository. Kindly add one using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1:**